File: ce/snippets/visualbasic/CRMV8/auditingvb/vb/auditing8.vb
' <snippetauditing8>


 Console.WriteLine("Retrieving audit details for an audit record.")

 ' Retrieve the audit details and display them.
 Dim auditDetailsRequest = New RetrieveAuditDetailsRequest With {.AuditId = auditSampleId}

 Dim auditDetailsResponse = CType(_service.Execute(auditDetailsRequest), RetrieveAuditDetailsResponse)

' </snippetauditing8>

File: ce/customer-engagement/on-premises/snippets/csharp/CRMV8/auditing/cs/auditing8.cs
// <snippetauditing8>


 Console.WriteLine("Retrieving audit details for an audit record.");

 // Retrieve the audit details and display them.
 var auditDetailsRequest = new RetrieveAuditDetailsRequest
 {
     AuditId = auditSampleId
 };

 var auditDetailsResponse =
     (RetrieveAuditDetailsResponse)_service.Execute(auditDetailsRequest);

// </snippetauditing8>